idunn/blocks/opening_hour.py

```diff
@@ -0,0 +1,34 @@
+import re
+import logging
+import datetime
+from apistar import types, validators
+
+class OpeningHourBlock(types.Type):
+    type = validators.String(title="type")
+    status = validators.String()
+    next_transition_time = validators.String()
+    seconds_before_next_transition = validators.Integer()
+    is_24_7 = validators.Boolean()
+    raw = validators.String()
+    days = validators.Array()
+
+    @classmethod
+    def from_es(cls, es_poi, lang):
+        logger = logging.getLogger('idunn')
+
+        raw = es_poi.get('properties', {}).get('opening_hours')
+        is247 = False
+        if raw is None:
+            return None
+        elif raw == '24/7':
+            is247 = True
+
+        return cls(
+            type='OpeningHourBlock',
```

Comment:
can't we get this from `cls` ?

Reply:
Also, the class name should not be used as `type`.
`opening_hours` is expected here in snake case (see the specs).